index.html

```diff
@@ -14,6 +14,10 @@
       })(window, document, "script", "dataLayer", "GTM-KB579ZN7");
     </script>
 
+    <style>
+      @import url("https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;0,900;1,400;1,700;1,900&display=swap");
+    </style>
+
```

Comment:
not sure I understand the fix: is it moving the reference into index.html, or is it those `wght@0,400,700...` numbers?

Reply:
It's moving the reference to `index.html`. Before, it was being handled in `main.tsx`, which I think caused React to escape or ignore the import.

Specifying the weights is optional but also decreases the size of the asset bundle Google Fonts will send over, since it won't try to load every font weight.